packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -282,6 +361,20 @@ export interface KeyProps {
    */
   readonly enabled?: boolean;
 
+  /**
+   * The cryptographic configuration of the key. The valid value depends on usage of the key.
```

Comment:
I think it's important to capture the same warning present in the docs here and for `keyUsage`:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-kms-key.html#cfn-kms-key-keyusage

Effectively a nice big "**WARNING** - Changing this property will..." in the docstring so it's clear that users can't arbitrarily change these.

Reply:
As I couldn't find docstrings of constructs using WARNING, I mentioned this with IMPORTANT.

Reply:
Sure, that works. Thanks!